src/components/component/navegation/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+import './navegation.css';
+
+const Navegation = () => (
+  <header>
+    <ul>
+      <li><NavLink to="/home" activeclass="active">HOME</NavLink></li>
```

Comment:
You can style any component with styledcomponents, avoid using classname.

Reply:
DONE